plugin.json

```diff
@@ -2,8 +2,8 @@
     "id": "com.mattermost.mscalendar",
     "name": "Microsoft Calendar",
     "description": "Microsoft Calendar Integration",
-    "version": "0.1.0",
-    "min_server_version": "5.16.0",
+    "version": "0.1.0-beta",
+    "min_server_version": "5.24.0",
```

Comment:
thought this was to stay 5.16 for the builds we make...

Reply:
I thought there was a bug with KVStore cron jobs sor something in prior to 5.19.1 that MS cal relies upon.